sdk/evaluation-context.json

```diff
@@ -23,6 +23,67 @@
             "title": "EnvironmentContext",
             "type": "object"
         },
+        "FeatureContext": {
+            "description": "Represents a feature context for feature flag evaluation.",
+            "properties": {
+                "key": {
+                    "description": "Key used when selecting a value for a multivariate feature. Set to an internal identifier or a UUID, depending on Flagsmith implementation.",
+                    "title": "Key",
+                    "type": "string"
+                },
+                "enabled": {
+                    "description": "Indicates whether the feature is enabled in the environment.",
+                    "title": "Enabled",
+                    "type": "boolean"
+                },
+                "value": {
+                    "description": "A default environment value for the feature. If the feature is multivariate, this will be the control value.",
+                    "title": "Value",
+                    "type": "string"
+                },
+                "variants": {
+                    "description": "An array of environment default values associated with the feature. Contains a single value for standard features, or multiple values for multivariate features.",
+                    "items": {
+                        "$ref": "#/$defs/FeatureValue"
+                    },
+                    "title": "Variants",
+                    "type": "array"
+                },
+                "priority": {
+                    "description": "Indicates the priority of the feature context. Higher values indicate higher priority in case when multiple feature contexts for the same feature are applicable",
```

Comment:
```suggestion
                    "description": "Indicates the priority of the feature context. Higher values indicate higher priority in case when multiple feature contexts for the same feature are applicable.",
```